commands/peers/addpeer.go

```diff
@@ -43,11 +50,65 @@ func addPeerHandler(w http.ResponseWriter, r *http.Request) {
 		rest.SendHTTPError(w, http.StatusInternalServerError, *rsp.OpError)
 		return
 	}
+
+	c := context.EtcdClient
+	// Add member to etcd server
+	mAPI := etcdclient.NewMembersAPI(c)
+	member, e := mAPI.Add(etcdcontext.Background(), "http://"+p.Name+":2380")
+	if e != nil {
+		log.WithField("err", e).Error("Failed to add member")
+		rest.SendHTTPError(w, http.StatusInternalServerError, e.Error())
+		return
+	}
+
+	newID := member.ID
+	newName := "name_" + p.Name
```

Comment:
A better name than "_name" required. Probably "ETCD_" would be a better fit here.